lib/compiler.js

```diff
@@ -110,3 +129,43 @@ function getCompilerName (context, filename) {
   var relativePath = path.relative(context, absolutePath);
   return 'html-webpack-plugin for "' + (absolutePath.length < relativePath.length ? absolutePath : relativePath) + '"';
 }
+
+function mergeChunks(thisChunk, otherChunk, reason) {
+  const otherChunkModules = otherChunk.modules.slice();
+	otherChunkModules.forEach(module => otherChunk.moveModule(module, thisChunk));
```

Comment:
can you fix the indent?
There is an `.editorconfig` on that project, that should take care of that for you